### PR TITLE
Final batch of text changes for transport

### PR DESCRIPTION
**Pull request by martinthomson**
Now I have to scrub the XML...

### CODE REVIEW

**Review by nibanks — APPROVED** [review]
LGTM.

File: draft-ietf-quic-transport.md [hunk 1]
```diff
@@ -6665,14 +6665,14 @@ Both on-path and off-path attackers can mount a passive attack in which they
 save observed packets for an offline attack against packet protection at a
 future time; this is true for any observer of any packet on any network.
 
-A blind attacker, one who injects packets without being able to observe valid
-packets for a connection, is unlikely to be successful, since packet protection
-ensures that valid packets are only generated by endpoints that possess the
-key material established during the handshake; see {{handshake}} and
-{{handshake-properties}}. Similarly, any active attacker that observes packets
+An attacker that injects packets without being able to observe valid packets for
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:6668]: good catch.

File: draft-ietf-quic-transport.md [hunk 2]
```diff
@@ -6881,7 +6882,7 @@ that are received on a different network path.
 The Source and Destination Connection ID fields are the primary means of
 protection against off-path attack during the handshake; see
 {{validate-handshake}}.  These are required to match those set by a peer.
-Except for an Initial and stateless reset packets, an endpoint only accepts
+Except for Initial and stateless reset packets, an endpoint only accepts
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:6885]:
```suggestion
Except for Initial and Stateless Reset packets, an endpoint only accepts
```

File: draft-ietf-quic-transport.md [hunk 3]
```diff
@@ -7613,13 +7605,13 @@ The initial contents of this registry are shown in {{iana-error-table}}.
 | 0x08  | TRANSPORT_PARAMETER_ERROR | Error in transport parameters | {{error-codes}} |
 | 0x09  | CONNECTION_ID_LIMIT_ERROR | Too many connection IDs received | {{error-codes}} |
 | 0x0a  | PROTOCOL_VIOLATION        | Generic protocol violation    | {{error-codes}} |
-| 0x0b  | INVALID_TOKEN             | Invalid Token Received        | {{error-codes}} |
+| 0x0b  | INVALID_TOKEN             | Invalid Token received        | {{error-codes}} |
```
Comment (janaiyengar) [inline draft-ietf-quic-transport.md:7608]:
```suggestion
| 0x0b  | INVALID_TOKEN             | Invalid token received        | {{error-codes}} |
```

**Review by janaiyengar — COMMENTED** [review]
Just a couple of suggestions.

**Review by janaiyengar — APPROVED** [review]

**Pull request merged**